### ISSUE TITLE
jruby: Can't load library installed via brew

### ISSUE THREAD

**Issue by HoneyryderChuck** [ffi/ffi#901]
After I install a lib via brew:

```
> brew install libidn2
> brew --prefix /usr/local/opt/libidn2
```

I load it like in this in my module via `ruby-ffi`:

```ruby
extend FFI::Library
ffi_lib ["libidn2", "libidn2.0"]
```

This code works when run under CRuby, but Jruby can't load the dynamic library:

```
 LoadError: Could not open library 'libidn2' : dlopen(libidn2, 5): image not found. Could not open library 'libidn2.dylib' : dlopen(libidn2.dylib, 5): image not found. Could not open library 'libidn2.0' :
dlopen(libidn2.0, 5): image not found. Could not open library 'libidn2.0.dylib' : dlopen(libidn2.0.dylib, 5): image not found
                                                                                              ffi_lib at /home/.rubies/jruby-9.2.17.0/lib/ruby/stdlib/ffi/library.rb:114
                                                                                                  map at org/jruby/RubyArray.java:2577
                                                                                              ffi_lib at /home/.rubies/jruby-9.2.17.0/lib/ruby/stdlib/ffi/library.rb:84
                                                                                         <module:Lib> at /home/dev/ruby-idn2/lib/idnx/idn2.rb:8
                                                                                        ...

```

I [found a similar issue reported for ruby-vips](https://github.com/libvips/ruby-vips/issues/284) but can't tell whether that's an acceptable workaround, as the jruby way should also reach the location where brew installed the lib.

**Comment by larskanis** [comment]
Which version of ffi on C ruby works? You might open an issue on the JRuby project, if not already done. Please also add the output of `ruby -v`.

**Comment by HoneyryderChuck** [comment]
The results can be observed [here](https://github.com/HoneyryderChuck/idnx/runs/2849724922?check_suite_focus=true). And it's `1.15.3 (java)`. 

I'll share with the JRuby folks then.

**Comment by HoneyryderChuck** [comment]
Closing, as the issue is fixed for jruby 9.3
